src/AVFoundation/Enums.cs

```diff
@@ -714,22 +722,32 @@ public enum AVContentAuthorizationStatus : long {
 		NotPossible,
 	}
 
-	[NoiOS][NoTV][NoWatch]
-	[Mac (10,10)]
+#if !NET
+	[iOS (15,0), TV (15,0), Watch (8,0), Mac (10,10), MacCatalyst (15,0)]
+#else
+	[SupportedOSPlatform ("ios15.0"), SupportedOSPlatform ("tvos15.0"), SupportedOSPlatform ("macos10.10"), SupportedOSPlatform ("maccatalyst15.0")]
```

Comment:
still present
not a big deal since, iirc, the generator will just ignore it since this is a processed file

```
AVFOUNDATION_API_SOURCES = \
	AVFoundation/Enums.cs \
```

which also means it did not require any availability changes ;-)